pkg/catalog/testutils/testutils.go

```diff
@@ -148,8 +149,15 @@ const (
 
 func (w *FakeWalker) createEntries(count int) {
 	ents := make([]store.ObjectStoreEntry, count)
+
+	// Use same sequence to overcome Graveler ability to create small ranges.
+	// Calling test functions rely on Graveler to not break on the first 1000 entries.
+	// For example, setting "5" here will cause the test to constantly fail.
+	// Fix Bug #3384
+	//nolint:gosec
+	randSeed := rand.New(rand.NewSource(6))
```

Comment:
This produce the same output every run, right?

Reply:
Right